pkg/resourcecollector/resourcecollector.go

```diff
@@ -724,8 +727,23 @@ func (r *ResourceCollector) pruneOwnedResources(
 					}
 				}
 			} else {
+				// skipOwnerRefCheck is set at a resource level
```

Comment:
Shouldn't we have "reset" the owner references for those objects if `skipOwnerReference` annotation was set on them? cc @siva-portworx 

Lets file a separate ticket to make this change

Reply:
As discussed, this is already being done.